pyiron_base/jobs/job/generic.py

```diff
@@ -632,10 +633,15 @@ def write_input(self):
         Call routines that generate the code specific input files
         Returns:
         """
-        write_input_files_from_input_dict(
-            input_dict=self.get_input_parameter_dict(),
-            working_directory=self.working_directory,
-        )
+        if not self._input_is_written:
+            write_input_files_from_input_dict(
+                input_dict=self.get_input_parameter_dict(),
+                working_directory=self.working_directory,
+            )
+        else:
+            self._logger.info(
+                "write_input was called, but input has already been written."
```

Comment:
```suggestion
                "write_input was called, but working_directory is not empty."
```

Reply:
added a check to the write_input_from_input_dict function